KappaAnalysis/interface/KappaEventProviderBase.h

```diff
@@ -36,7 +36,8 @@ class KappaEventProviderBase: public EventProviderBase<TTypes> {
 		m_fi.SpeedupTree(128*1024*1024); // in units of bytes
 
 		// auto-delete objects when moving to a new object. Not default root behaviour
-		m_fi.eventdata.SetAutoDelete(true);
```

Comment:
why has this been turned on? I couldn't find anything useful in the corresponding commit message

Mind that github can't really follow the changes done here since the file has been renamed twice. git itself on the other hand can follow.